tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py

```diff
@@ -30,52 +33,20 @@ def _configure_plugin(self, plugin_name, plugin_params):
                 "as conf parameters" % (
                     plugin_name,
                     NS.node_context.fqdn,
-                    plugin_params
+                    json.dumps(plugin_params).encode('utf-8')
                 )
             },
             job_id=self.parameters['job_id'],
             flow_id=self.parameters['flow_id']
         )
-        cmd_str = "tendrl_monitoring_config_manager %s '%s'" % (
-            plugin_name,
-            plugin_params
+        subprocess.call(
```

Comment:
use try:
subprocess.check_call(args...)
except subprocess.CalledProcessError as ex:
    log the event and raise ex